Panic creating a module outside of an FTL project

```
mtoohey ~/Code  $ ./ftl/build/release/ftl new go . example
panic: project config path must be absolute: 

goroutine 1 [running]:
github.com/TBD54566975/ftl/internal/projectconfig.Config.Root({{0x0, 0x0}, {0x0, 0x0}, {0x0, 0x0}, 0x0, {0x0, 0x0, 0x0}, ...})
	/Users/mtoohey/Code/ftl/internal/projectconfig/projectconfig.go:46 +0x94
main.newCmd.Run({{0x140002fa4f0, 0x2}, {0x10315b8d0, 0x1}, {0x140002fa530, 0x7}}, {0x1022885b8, 0x14000715920}, 0x140008ae080, {{0x0, ...}, ...})
	/Users/mtoohey/Code/ftl/frontend/cli/cmd_new.go:109 +0x3d0
reflect.Value.call({0x1020b7be0?, 0x103343db8?, 0x14000a898f8?}, {0x101594259, 0x4}, {0x14000cbf440, 0x3, 0x100a195d8?})
	/Users/mtoohey/Library/Caches/hermit/pkg/go-1.23.2/src/reflect/value.go:581 +0x97c
reflect.Value.Call({0x1020b7be0?, 0x103343db8?, 0x14000a899f8?}, {0x14000cbf440?, 0x102256a20?, 0x14000a89930?})
	/Users/mtoohey/Library/Caches/hermit/pkg/go-1.23.2/src/reflect/value.go:365 +0x94
github.com/alecthomas/kong.callFunction({0x1020b7be0?, 0x103343db8?, 0x14000a89b58?}, 0x14000715a70)
	/Users/mtoohey/go/pkg/mod/github.com/alecthomas/kong@v1.2.1/callbacks.go:98 +0x390
github.com/alecthomas/kong.(*Context).RunNode(0x140008ae080, 0x14000a2a2d0, {0x14000a89e78, 0x1, 0x140006ee140?})
	/Users/mtoohey/go/pkg/mod/github.com/alecthomas/kong@v1.2.1/context.go:798 +0x604
github.com/alecthomas/kong.(*Context).Run(0x1022885b8?, {0x14000a89e78?, 0x140008ae080?, 0x1032f2db0?})
	/Users/mtoohey/go/pkg/mod/github.com/alecthomas/kong@v1.2.1/context.go:823 +0x138
main.main()
	/Users/mtoohey/Code/ftl/frontend/cli/main.go:139 +0x5d4
```

--- a/frontend/cli/cmd_new.go
+++ b/frontend/cli/cmd_new.go
@@ -55,7 +55,19 @@ func prepareNewCmd(ctx context.Context, k *kong.Kong, args []string) (optionalPl
 		return optionalPlugin, fmt.Errorf("could not find new command")
 	}
 
-	plugin, err := languageplugin.New(ctx, pluginDir(), language, "new")
+	projConfigPath, ok := projectconfig.DefaultConfigPath().Get()
+	if !ok {
+		return optionalPlugin, fmt.Errorf("could not find project config path")
+	}
+	_, err = projectconfig.Load(ctx, projConfigPath)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return optionalPlugin, fmt.Errorf(`could not find FTL project config: try running "ftl init"`)
+		}
+		return optionalPlugin, fmt.Errorf("could not load project config: %w", err)
+	}
+
+	plugin, err := languageplugin.New(ctx, filepath.Dir(projConfigPath), language, "new")
 	if err != nil {
 		return optionalPlugin, fmt.Errorf("could not create plugin for %v: %w", language, err)
 	}
@@ -78,15 +90,6 @@ func prepareNewCmd(ctx context.Context, k *kong.Kong, args []string) (optionalPl
 	}
 	newCmdNode.Flags = append(newCmdNode.Flags, flags...)
 	return optional.Some(plugin), nil
-}
-
-// pluginDir returns the directory to base the plugin working directory off of.
-// It tries to find the root directory of the project, or uses "." as a fallback.
-func pluginDir() string {
-	if configPath, ok := projectconfig.DefaultConfigPath().Get(); ok {
-		return filepath.Dir(configPath)
-	}
-	return "."
 }
 
 func (i newCmd) Run(ctx context.Context, ktctx *kong.Context, config projectconfig.Config, plugin *languageplugin.LanguagePlugin) error {

